frontend/css/year-in-music.less

```diff
@@ -568,7 +568,8 @@
       overflow: hidden;
       .graph {
         color: var(--selectedColor);
-        overflow: auto;
+        overflow: scroll;
```

Comment:
I think this should allow  scrolling horizontally when needed (mobile sizes), while not showing the scrollbar when unnecessary:
```suggestion
        overflow-x: auto;
```

Reply:
Done!